flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/alias/ClearLookupJoinHintWithInvalidPropagationShuttleTest.java

```diff
@@ -120,22 +125,53 @@ public void testClearLookupHintWithInvalidPropagationToViewWhileViewHasLookupHin
     public void testClearLookupHintWithInvalidPropagationToSubQuery() {
         // SELECT /*+ LOOKUP('table'='lookup', 'retry-predicate'='lookup_miss',
         // 'retry-strategy'='fixed_delay', 'fixed-delay'='155 ms', 'max-attempts'='10',
-        // 'async'='true', 'output-mode'='allow_unordered','capacity'='1000', 'time-out'='300 s' */*
-        //   FROM (
-        //     SELECT src.a
-        //     FROM src
-        //     JOIN lookup FOR SYSTEM_TIME AS OF T.proctime AS D
-        //       ON T.a = D.id
-        //   ) t1 JOIN src t2 ON t1.a = t2.a
+        // 'async'='true', 'output-mode'='allow_unordered','capacity'='1000', 'time-out'='300 s')
+        // */ t1.a
+        //  FROM (
+        //      SELECT s.a
+        //      FROM src s
+        //      JOIN lookup FOR SYSTEM_TIME AS OF s.pts AS d
+        //      ON s.a=d.a
+        //  ) t1
+        //  JOIN src t2
+        //  ON t1.a=t2.a
+
+        CorrelationId cid = builder.getCluster().createCorrel();
+        RelDataType aType = builder
+                .getTypeFactory()
+                .createStructType(
+                        Arrays.asList(builder.getTypeFactory().createSqlType(SqlTypeName.BIGINT)),
+                        Arrays.asList("a")
```

Comment:
nit: 'Arrays.asList' on single element can be replaced with 'Collections.singletonList'